docs/upgrade_tails_usbs.rst

```diff
@@ -0,0 +1,124 @@
+Updating Tails USBs
+===================
+
+There are 3 components of SecureDrop that use Tails USBs:
+
+  #. The *Admin Workstation* (allowed to connect to the Internet)
+  #. The *Journalist Workstation* (allowed to connect to the Internet)
+  #. The *Secure Viewing Station* (airgapped, never allowed to connect to any network)
+
+You are responsible for keeping your Tails USBs updated. This guide will
+explain that process.
+
+One-Click Updates
+-----------------
+
+For the Internet-connected Tails workstations (the *Admin Workstation* and
+the *Journalist Workstation*), you'll see a notification when a new version
+of Tails is available. We recommend that you :ref:`back up your existing
+configuration<Step 1>` and upgrade as soon as possible.
+
+|Upgrade Notification|
+
+It usually takes some time for updates to download, so keep that in mind when choosing when to upgrade.
+
+.. |Upgrade Notification| image:: images/tails_upgrade_notification.png
+
+
+Manual Updates
+--------------
+
+However, because the *Secure Viewing Station (SVS)* is airgapped, it cannot
```

Comment:
For flow, I would simply start with "Because the .." (we will likely often link directly to this section).

Reply:
Fixed in c91e4931f